### PR TITLE
fix: update generate endpoints function

### PR DESCRIPTION
**Pull request by metalurgical**

### CODE REVIEW

**Comment by metalurgical** [comment]
Resolve #69 
Close #69

**Comment by metalurgical** [comment]
Resolve #64 
Close #64

File: packages/tss-client/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "author": "Web3Auth",
   "license": "ISC",
   "dependencies": {
-    "@toruslabs/eccrypto": "^4.0.0",
+    "@toruslabs/eccrypto": "^5.0.4",
     "@types/chrome": "^0.0.268",
```
Comment (himanshuchawla009) [inline packages/tss-client/package.json:24]: this types package should be a dev dependency

**Review by himanshuchawla009 — DISMISSED** [review]

File: packages/tss-client/package.json (hunk 1 above)
Comment (metalurgical) [reply packages/tss-client/package.json]: It has been moved.

**Review by himanshuchawla009 — APPROVED** [review]

**Pull request merged**